binaries/cuprated/src/main.rs

```diff
@@ -61,6 +61,9 @@ fn main() {
     // Initialize logging.
     logging::init_logging(&config);
 
+    // Logging Config
+    println!("{config}");
```

Comment:
Because the display impl wasn't returning anything to write (it was writing it self) this was just printing a blank line. Changed so the config gets output to the logging interface.

```suggestion
    info!("{config}");
```